labels/de-DE.json

```diff
@@ -369,7 +369,7 @@
   "orderNo": "Order #",
   "orderNumber": "Bestellnummer",
   "orderPickupBlob": "Bonne nouvelle! <br/> Votre commande est disponible et prête à être récupérée. Veuillez cliquer sur le bouton «Je suis en route» ci-dessous pour informer le magasin que vous êtes sur le point de récupérer votre commande. Merci beaucoup pour votre achat! nous sommes heureux de pouvoir vous aider à obtenir ce que vous recherchez.",
-  "orderPickupReminderTitle": "RAPPEL: VOTRE COMMANDE EST PRÊT POUR LE RETRAIT!", 
+  "orderPickupReminderTitle": "RAPPEL: VOTRE COMMANDE EST PRÊT POUR LE RETRAIT!",
```

Comment:
Remove this change from PR. since you have not changed this label.

Reply:
Done